assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -21,6 +21,10 @@ che.api=http://${CHE_HOST}:${CHE_PORT}/api
 # for websocket interaction/messaging.
 che.websocket.endpoint=ws://${CHE_HOST}:${CHE_PORT}/api/websocket
 
+# Che websocket minor endpoint. Provides basic communication endpoint
```

Comment:
What if we just introduce new endpoint for logs and will not use `major`, `minor` in endpoints names? For me, `minor`/`major` looks not self-explained at all.

Reply:
It's actually not about the logs, the naming comes from the degree of significance of messages that are transmitted over an endpoint. "Major" endpoint is expected to receive important messages, while "minor" is for the rest (e.i. logs). However I would like to know if it is possible to make it more clear, what is your suggestion ?

Reply:
> while "minor" is for the rest (e.i. logs)

Do we have something else for minor endpoint?

I suggest consider having `Che WebSocket Endpoint`(major) and `Che Logs WebSocket Endpoint`(minor). If we would have something else for minor endpoint - then it makes sense to introduce new endpoint and be able to configure `messages_pull_size` separately.

Reply:
I don't have strong feelings about that. So, up to the author of the PR if there are no objections from other sites